.circleci/config.yml

```diff
@@ -48,7 +48,47 @@ jobs:
           command: |
             ./project.rb circle-deploy \
               --project all-of-us-workbench-test --creds-file circle-sa-key.json
-
+  api-build-integration-test:
+    docker:
+      - image: allofustest/workbench:buildimage-0.0.10
+    working_directory: ~/workbench
+    environment:
+      JVM_OPTS: -Xmx3200m
+      TERM: dumb
+    steps:
+      - checkout
+      - run:
+          name: Fetch Submodules
+          command: git submodule update --init
+      - restore_cache:
+          keys:
+          - api-cache-{{ checksum "~/workbench/api/build.gradle" }}
+          - api-cache-
+      - run:
+          name: Validate swagger definitions
```

Comment:
Does this need to be an explicit step here?

Reply:
I don't think so? Because we'll get alerts if our swagger is broken from our standard API tests?

Reply:
Correct. We can remove this.